installer/templates/phx_umbrella/apps/app_name/lib/app_name/application.ex

```diff
@@ -6,9 +6,13 @@ defmodule <%= app_module %>.Application do
   use Application
 
   def start(_type, _args) do<%= if ecto do %>
-    Supervisor.start_link([
-      <%= app_module %>.Repo,
-    ], strategy: :one_for_one, name: <%= app_module %>.Supervisor)<% else %>
```

Comment:
What if we do this:

```
children = [

]

Supervisor.start_link(children, strategy: :one_for_one, name: ...
```

?

In case Ecto is not available, we can do this for simplicity:

```elixir
children = [
  # MyApp.Worker
]

Supervisor.start_link(children, ....)
```